docs/source/traits_api_reference/has_traits.rst

```diff
@@ -41,7 +41,6 @@ Classes
 
 .. autoclass:: ISerializable
 
-.. autoclass:: traits_super
```

Comment:
Do we want to lose another blank line here?

(I had to find _something_ to complain about. :-)

Reply:
Yes! (Who knows if I deliberately left it there for it to be complained about ;)  )